test/runtests.jl

```diff
@@ -1,207 +1,385 @@
 using Test, UUIDs
 using PackageAnalyzer
-using PackageAnalyzer: parse_project
+using PackageAnalyzer: parse_project, Release, Added, Dev, Trunk, PkgSource
 using JLLWrappers
-using GitHub
+using GitHub: GitHub
```

Comment:
What's the point of this?

Reply:
```
WARNING: using GitHub.Release in module Main conflicts with an existing identifier.
```

Reply:
I guess my question is: how's this different from `import GitHub`?

Reply:
Oh, it’s not. YAS style guide requires this form so I am used to writing it, but it’s the same

Reply:
Uhm, I find it a bit bizarre to be honest, but it's ok I guess.

Reply:
We can change it, I don’t care either way:
```suggestion
import GitHub
```